qsutil.h

```diff
@@ -2,7 +2,10 @@
 #define QSUTIL_H
 
 extern void log1();
-extern void log2();
+extern void qslog2();
+#ifdef QMAIL_COMPAT_LOG2
```

Comment:
```c
#ifndef DONT_PROVIDE_OBSOLETE
```
as in ```alloc.h```?

Reply:
I knew we'd done this somewhere before and couldn't remember where!